.clang-format

```diff
@@ -16,4 +16,3 @@ BinPackParameters: false
 AlwaysBreakTemplateDeclarations: Yes
 ReflowComments: false
 BinPackArguments: false
-BinPackParameters: false
```

Comment:
Are these updates related to using newer `clang`?

Reply:
The `BinPackParameters` was just a duplicate entry. `SortIncludes` is a change in `clang-13`, since the Makefile specifically uses `clang-format-10` I can revert that change.

Reply:
Yeah, maybe we could update the clang version used for formatting at some point.